test/bridge.test.ts

```diff
@@ -421,16 +413,16 @@ describe("Bridge", async function () {
     const l2RewardsIndexUsdc = uintFromParts(flushL2Messages[1].payload[6], flushL2Messages[1].payload[7]);
 
     // actually withdraw tokens
-    txDai = await l1Bridge.connect(l1user).withdraw(aDai.address, l2user.starknetContract.address, l1user.address, 28, l2RewardsIndexDai, false);
+    txDai = await l1Bridge.connect(l1user).withdraw(aDai.address, l2user.starknetContract.address, l1user.address, 27n * DAI_UNIT, l2RewardsIndexDai, false);
     blockNumberDai = txDai.blockNumber;
-    txUsdc = await l1Bridge.connect(l1user).withdraw(aUsdc.address, l2user.starknetContract.address, l1user.address, 37, l2RewardsIndexUsdc, false);
+    txUsdc = await l1Bridge.connect(l1user).withdraw(aUsdc.address, l2user.starknetContract.address, l1user.address, 37n * USDC_UNIT, l2RewardsIndexUsdc, false);
     blockNumberUsdc = txUsdc.blockNumber;
 
     // check that tokens have been transfered to l1user
-    expect(await aDai.balanceOf(l1user.address)).to.equal(l1InitialADaiBalance);
-    expect(await aUsdc.balanceOf(l1user.address)).to.equal(l1InitialAUsdcBalance);
-    expect(await aDai.balanceOf(l1Bridge.address)).to.equal(30);
-    expect(await aUsdc.balanceOf(l1Bridge.address)).to.equal(40);
+    expect(BigNumber.from(l1InitialADaiBalance)).to.be.lte(BigNumber.from(await aDai.balanceOf(l1user.address)));
```

Comment:
Are we using  `to.be.lte` because the balance of aTokens in `l1user` wallet increases? -just double checking :)

Reply:
yes, that's the exact reason. when using smaller amounts, the amount was basically the same everytime. now that we use bigger amount, the decimals change at each run due to atoken rounding errors. this is why we use inequalities instead